src/qml/PatchManagerPage.qml

```diff
@@ -473,16 +491,34 @@ Page {
                     busy: patchObject.busy
                 }
 
-                Label {
+                Column {
                     id: nameLabel
                     anchors.left: appliedSwitch.right
                     anchors.right: patchIcon.status == Image.Ready ? patchIcon.left : parent.right
                     anchors.margins: Theme.paddingMedium
                     anchors.verticalCenter: parent.verticalCenter
-                    text: name
-                    color: patchObject.details.isCompatible ? background.down ? Theme.highlightColor : Theme.primaryColor
-                                                            : background.down ? Qt.tint(Theme.highlightColor, "red") : Qt.tint(Theme.primaryColor, "red")
-                    truncationMode: TruncationMode.Fade
+                    Label {
+                        text: name
+                        color: patchObject.details.isCompatible ? background.down ? Theme.highlightColor : Theme.primaryColor
+                                                                : background.down ? Qt.tint(Theme.highlightColor, "red") : Qt.tint(Theme.primaryColor, "red")
```

Comment:
(unrelated) wondering if we should use some [alpha](https://doc.qt.io/archives/qt-5.6/qml-qtqml-qt.html#tint-method) on this color tinting